src/app/Http/Controllers/Auth/ForgotPasswordController.php

```diff
@@ -29,7 +30,9 @@ class ForgotPasswordController extends Controller
      */
     public function __construct()
     {
-        $this->middleware('guest');
+        $guard = config('backpack.base.guard') ? config('backpack.base.guard') : config('auth.defaults.guard');
```

Comment:
I would've kept it like this: `config('backpack.base.guard') ?: config('auth.defaults.guard')`